bitbots_hcm/bitbots_hcm/hcm_dsd/hcm_blackboard.py

```diff
@@ -4,142 +4,112 @@
-        self.falling_animation_right = self.node.get_parameter("animations.falling_right").value
-        self.stop_animation = self.node.get_parameter("animations.penalty").value
-        self.sit_down_animation = self.node.get_parameter("animations.sit_down").value
-        self.motor_off_animation = self.node.get_parameter("animations.motor_off").value
-        self.stand_up_front_animation = self.node.get_parameter("animations.stand_up_front").value
-        self.stand_up_back_animation = self.node.get_parameter("animations.stand_up_back").value
-        self.stand_up_left_animation = self.node.get_parameter("animations.stand_up_left").value
-        self.stand_up_right_animation = self.node.get_parameter("animations.stand_up_right").value
-        # motors
-        # initialize with current time, or motors will be turned off on start
-        self.last_motor_goal_time = self.node.get_clock().now()
+        self.external_animation_running: bool = False
+        self.last_animation_goal_time: Time = self.node.get_clock().now()
+        self.record_active: bool = False
+
+        # Get animation parameters
+        self.animation_name_falling_front: str = self.node.get_parameter("animations.falling_front").value
+        self.animation_name_falling_back: str = self.node.get_parameter("animations.falling_back").value
+        self.animation_name_falling_left: str = self.node.get_parameter("animations.falling_left").value
+        self.animation_name_falling_right: str = self.node.get_parameter("animations.falling_right").value
+        self.animation_name_turning_back_left: str = self.node.get_parameter("animations.turning_back_left").value
+        self.animation_name_turning_back_right: str = self.node.get_parameter("animations.turning_back_right").value
+
+        # Motor State
+        # Initialize with current time, or motors will be turned off on start
+        self.last_motor_goal_time: Time = self.node.get_clock().now()
         self.last_motor_update_time =  Time()
-        self.motor_timeout_duration = self.node.get_parameter("motor_timeout_duration").value
-        self.motor_off_time = self.node.get_parameter("motor_off_time").value
-        self.imu_timeout_duration = self.node.get_parameter("imu_timeout_duration").value
         self.current_joint_state: Optional[JointState] = None
         self.previous_joint_state: Optional[JointState] = None
-        anim_package = self.node.get_parameter("animations.anim_package").value
-        path = get_package_share_directory(anim_package)
-        path = os.path.join(path, 'animations/motion/', self.walkready_animation + '.json')
-        with open(path, 'r') as f:
-            json_data = json.load(f)
-        keyframes: List[dict] = json_data["keyframes"]
-        self.walkready_pose_dict: dict = keyframes[-1]["goals"]
-        self.walkready_pose_threshold = self.node.get_parameter("animations.walkready_pose_threshold").value
```

Comment:
I hope, this is now handled by the dynup or animation server?

Reply:
The dynup is not fully responsible for the walkready pose.

Reply:
I was just asking because this code part was removed

Reply:
This was a autocomplete error. I meant to say
"The dynup is **now** fully responsible for the walkready pose."

Reply:
Therefore this can be removed